core/chains/evm/config/chain_specific_config.go

```diff
@@ -90,39 +91,39 @@ func setChainSpecificConfigDefaultSets() {
-		gasBumpWei:                                 *assets.GWei(5),
-		gasEstimatorMode:                           "BlockHistory",
-		gasLimitDefault:                            DefaultGasLimit,
-		gasLimitMultiplier:                         1.0,
-		gasLimitTransfer:                           21000,
-		gasPriceDefault:                            *DefaultGasPrice,
-		gasTipCapDefault:                           *DefaultGasTip,
-		gasTipCapMinimum:                           *big.NewInt(0),
-		headTrackerHistoryDepth:                    100,
-		headTrackerMaxBufferSize:                   3,
-		headTrackerSamplingInterval:                1 * time.Second,
-		layer2Type:                                 "",
-		linkContractAddress:                        "",
-		logBackfillBatchSize:                       100,
-		maxGasPriceWei:                             *assets.GWei(5000),
-		maxInFlightTransactions:                    16,
-		maxQueuedTransactions:                      250,
-		minGasPriceWei:                             *assets.GWei(1),
-		minIncomingConfirmations:                   3,
-		minRequiredOutgoingConfirmations:           12,
-		minimumContractPayment:                     DefaultMinimumContractPayment,
-		nonceAutoSync:                              true,
-		ocrContractConfirmations:                   4,
-		rpcDefaultBatchSize:                        100,
-		set:                                        true,
+		chainType:                        "",
+		eip1559DynamicFees:               false,
+		ethTxReaperInterval:              1 * time.Hour,
+		ethTxReaperThreshold:             168 * time.Hour,
+		ethTxResendAfterThreshold:        1 * time.Minute,
+		finalityDepth:                    50,
+		gasBumpPercent:                   20,
+		gasBumpThreshold:                 3,
+		gasBumpTxDepth:                   10,
+		gasBumpWei:                       *assets.GWei(5),
+		gasEstimatorMode:                 "BlockHistory",
+		gasLimitDefault:                  DefaultGasLimit,
+		gasLimitMultiplier:               1.0,
+		gasLimitTransfer:                 21000,
+		gasPriceDefault:                  *DefaultGasPrice,
+		gasTipCapDefault:                 *DefaultGasTip,
+		gasTipCapMinimum:                 *big.NewInt(0),
+		headTrackerHistoryDepth:          100,
+		headTrackerMaxBufferSize:         3,
+		headTrackerSamplingInterval:      1 * time.Second,
+		linkContractAddress:              "",
+		logBackfillBatchSize:             100,
+		maxGasPriceWei:                   *assets.GWei(5000),
+		maxInFlightTransactions:          16,
+		maxQueuedTransactions:            250,
+		minGasPriceWei:                   *assets.GWei(1),
+		minIncomingConfirmations:         3,
+		minRequiredOutgoingConfirmations: 12,
+		minimumContractPayment:           DefaultMinimumContractPayment,
+		nonceAutoSync:                    true,
+		ocrContractConfirmations:         4,
+		rpcDefaultBatchSize:              100,
+		set:                              true,
```

Comment:
I noticed this was unused. Leftover from development to be removed? Or something to wire up a check for?

Reply:
It used to be used to make sure that we weren't accidentally using the empty/zero struct. Worth double checking to make sure we aren't still using it for that reason.

Reply:
Added sanity check to end of `init()`.